mne_connectivity/spectral/time.py

```diff
@@ -82,6 +85,9 @@ def spectral_connectivity_time(data, method='coh', average=False,
         is equivalent to no smoothing.
     sm_kernel : {'square', 'hanning'}
         Smoothing kernel type. Choose either 'square' or 'hanning'.
+    padding : float
```

Comment:
Sorry maybe I missed this, but it's not clear to me what padding is meant to do. Do you mind adding a bit more info here in the docstring, or the `Notes` section?